.gitignore

```diff
@@ -8,6 +8,8 @@ __pycache__/
 
 # Distribution / packaging
 .Python
+.python-version
```

Comment:
Why this? Shouldn't we all agree on which python version to run?

Reply:
Kommer från pyenv, som är ett alternativ till (bla) virtualenv. Om vi börjar hårdkoda virtualenv-namnet kan jag tänka mig att vi inte skulle behöva denna :P

Reply:
Det är för att pyvenv (inte pyenv) är trasig. Jag har en plugin till pyenv som fixar detta, dock skriven i fish.